### PR TITLE
change validationmethod md for python

### PR DESCRIPTION
**Pull request by zhangxiaoli73**
## What changes were proposed in this pull request?

As validation method is changed in python, so have to update the md doc.

### CODE REVIEW

File: docs/docs/APIdocs/Metrics/ValidationMethod.md [hunk 1]
```diff
@@ -6,24 +6,30 @@ The trait can be extended by user-defined method. Now we have defined Top1Accura
...
 
 #### Loss ####
 
 Calculate loss of output and target with criterion. The default criterion is ClassNLLCriterion.
-
-val loss = new Loss(criterion)
+**Scala:**
+```val loss = new Loss(criterion)```
+**Python:**
+```loss = Loss(cri)```
```
Comment (zhichao-li) [inline docs/docs/APIdocs/Metrics/ValidationMethod.md]: add the default value for cri which is ClassNLLCriterion

Comment (zhangxiaoli73) [inline docs/docs/APIdocs/Metrics/ValidationMethod.md]: I explain the default value in the description.

**Comment by zhichao-li** [comment]
LGTM
